### PR TITLE
Feature/az deploy

### PR DESCRIPTION
**Pull request by suneetnangia**
Following are part of this PR:
1. Azure deployment of full solution with a single cmd line on Azure Shell (building on top of @buzzfrog's Bicep/PoSh scripts)
2. Addition do deployment design for solution in Readme.
3. Updates for some scripts to remove unwanted access levels on Event Hub.
4. Deploying platform components in edge-core NS of Kubernetes, and app to edge-app1 NS.

### CODE REVIEW

File: deployment/deploy-az-demo-bootstrapper.ps1 [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+./deploy-core-infrastructure.ps1 -ApplicationName $ApplicationName
+
+clear
+./deploy-core-platform.ps1 -ApplicationName $ApplicationName
+
+clear
+./deploy-app.ps1 -ApplicationName $ApplicationName -AKSClusterResourceGroupName $env:RESOURCEGROUPNAME -AKSClusterName $env:AKSCLUSTERNAME
+
+clear
+Write-Host "-------------------------------------------------------------------------------------------------------------------------------------------" -ForegroundColor Yellow
```
Comment (buzzfrog) [inline deployment/deploy-az-demo-bootstrapper.ps1:35]: Maybe you should use Write-Title here

File: deployment/deploy-az-demo-bootstrapper.ps1 [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+#  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+# ------------------------------------------------------------
+Param(
+    [string]
+    [Parameter(mandatory=$true)]
+    $ApplicationName
+)
+
+# ----- Copy scripts from source location
+$baseLocation = "https://raw.githubusercontent.com/suneetnangia/distributed-az-edge-framework/feature/docs-update"
```
Comment (buzzfrog) [inline deployment/deploy-az-demo-bootstrapper.ps1]: Is this really the right url?

Comment (suneetnangia) [reply deployment/deploy-az-demo-bootstrapper.ps1]: No, once I finish the testing (need to verify OPCUA telemetry), last update to this branch will be to change the url.

File: deployment/deploy-az-demo-bootstrapper.ps1 (hunk 1 above)
Comment (suneetnangia) [reply deployment/deploy-az-demo-bootstrapper.ps1:35]: Yeah, I think if we are considering to use Write-Title in multiple script files, we should may be look at moving that function to a common/shared module or something?

File: deployment/deploy-az-demo-bootstrapper.ps1 (hunk 2 above)
Comment (suneetnangia) [reply deployment/deploy-az-demo-bootstrapper.ps1]: Changing PR to draft to signify WIP.

File: deployment/deploy-az-demo-bootstrapper.ps1 (hunk 1 above)
Comment (suneetnangia) [reply deployment/deploy-az-demo-bootstrapper.ps1:35]: Will take a look at having a shared module for common functions in PoSh.

**Pull request merged**